tests/notification/test_notification.py

```diff
@@ -0,0 +1,118 @@
+"""Module containing the Notification test files for autopush-rs."""
+
+import logging
+import time
+from typing import Any
+
+import imgcompare
+import pytest
+from PIL import ImageGrab
+from selenium.webdriver.common.by import By
+
+
+@pytest.fixture
+def images_dir(tmpdir: Any) -> Any:
+    """Directory to store the screenshots for testing."""
+    return tmpdir.mkdir("images")
+
+
+@pytest.fixture(autouse=True)
+def setup_page(selenium: Any, images_dir: str) -> ImageGrab:
+    """Fixture to setup the test page and take the base screenshot."""
+    selenium.get("localhost:8201")
+    selenium.find_element(By.CSS_SELECTOR, ".container").click()
+    time.sleep(5)  # wait a bit to take the base screenshot
```

Comment:
Do you think a selenium explicit wait could potentially replace this sleep? I'm fine with the sleep for now (I'm guessing if the explicit wait is possible it might take some effort to determine which element to wait on) but if so, might be worth including a comment about that potential improvement

Reply:
This is outside of the selenium context, I am just trying to prevent getting a screenshot with no notification by taking it immediately after I hit click because we know computers will do it quite fast and that's unrealistic.